addons/mail/static/src/components/dialog/dialog.scss

```diff
@@ -12,6 +12,10 @@
     align-items: center;
     justify-content: center;
     z-index: 1;
+
+    &.o_dialog_mobile {
+        z-index: 1001; // above the chat window manager.
```

Comment:
Possible to use math from a variable instead of a random value?
`$o-mail-thread-window-zindex`, `$zindex-modal`, etc